clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerNetworkThreadTest.java

```diff
@@ -170,7 +173,8 @@ public void testSyncCommitEvent() {
     @Test
     public void testListOffsetsEventIsProcessed() {
         Map<TopicPartition, Long> timestamps = Collections.singletonMap(new TopicPartition("topic1", 1), 5L);
-        ApplicationEvent e = new ListOffsetsEvent(timestamps, true);
+        Timer timer = time.timer(100);
```

Comment:
And again. I don't think you're actually trying to bound the event's time here, but you have to provide a timer. I reckon `Long.MAX_VALUE` is better aligned with the spirit of this event.

Reply:
Is our usage of `ListOffsetsEvent` any different from how we're using events in this test? Why would a value of `100` not work?